etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverterTest.groovy

```diff
@@ -169,4 +175,114 @@ class HapiOrderConverterTest extends Specification {
+        entryList.add(patientEntry)
+        mockOrderBundle.setEntry(entryList)
+
+        when:
+        def convertedOrderBundle = HapiOrderConverter.getInstance().addContactSectionToPatientResource(mockOrder).getUnderlyingOrder() as Bundle
+
+        then:
+        def convertedPatient = convertedOrderBundle.getEntry().get(0).getResource() as Patient
+        def contactSection = convertedPatient.getContact()[0]
+
+        contactSection != null
+        // Name
+        def convertedPatientHumanName =
+                convertedPatient.castToHumanName(convertedPatient.getExtensionByUrl(patientMothersMaidenNameURL)
+                .getValue())
+        def contactSectionHunanName = contactSection.getName()
+        contactSectionHunanName.getText() == convertedPatientHumanName.getText()
+        contactSectionHunanName.getFamily() == convertedPatientHumanName.getFamily()
+        contactSectionHunanName.getGiven() == convertedPatientHumanName.getGiven()
+        // Relationship
+        def relationship = contactSection.getRelationship().get(0).getCoding().get(0)
+        relationship.getCode() == relationshipCode
+        relationship.getSystem() == relationshipSystem
+        relationship.getDisplay() == relationshipDisplay
+        // Telecom
+        def contactTelecom = contactSection.getTelecom().get(0)
+        def convertedPatientTelecom = convertedPatient.getTelecom().get(0)
+        contactTelecom.getSystem() == convertedPatientTelecom.getSystem()
+        contactTelecom.getValue() == convertedPatientTelecom.getValue()
+        contactTelecom.getUse() == convertedPatientTelecom.getUse()
+        // Address
+        def contactSectionAddress = contactSection.getAddress()
+        def convertedPatientAddress = convertedPatient.getAddress().get(0)
+        contactSectionAddress.getLine().get(0) == convertedPatientAddress.getLine().get(0)
+        contactSectionAddress.getCity() == convertedPatientAddress.getCity()
+        contactSectionAddress.getPostalCode() == convertedPatientAddress.getPostalCode()
+    }
+
+    def "no humanName section in contact"() {
+        given:
+        def addHumanName = false
+        def patientResourceNoHumanName = fakePatientResource(addHumanName)
+        def patientEntry = new Bundle.BundleEntryComponent().setResource(patientResourceNoHumanName)
+        def entryList = new ArrayList<Bundle.BundleEntryComponent>()
+        entryList.add(patientEntry)
+        mockOrderBundle.setEntry(entryList)
+        when:
+        def convertedOrderBundle = HapiOrderConverter.getInstance().addContactSectionToPatientResource(mockOrder).getUnderlyingOrder() as Bundle
+
+        then:
+        def convertedPatient = convertedOrderBundle.getEntry().get(0).getResource() as Patient
+        def contactSection = convertedPatient.getContact()[0]
+        def humanNameExtension = contactSection.getExtensionByUrl("http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName")
+
+        humanNameExtension == null
```

Comment:
The mother's maiden name extension is never on the contact, so this check doesn't make sense.  We  also wouldn't check for existence of that extension for this test.  If I'm reading the code in `OrderConverter` correctly, if the mother's maiden name extension under the patient doesn't exist, then the patient's contact's name doesn't exist.  So, then this test should check that the name of the contact is null, not the extension of the contact is null.